projects/spectator/src/lib/base/dom-spectator.ts

```diff
@@ -175,6 +175,30 @@ export abstract class DomSpectator<I> extends BaseSpectator {
     return event;
   }
 
+  public triggerEventHandler<C = any, K extends keyof C = any>(
+    directiveOrSelector: Type<C> | string | DebugElement,
+    eventName: K,
+    eventObj: EventEmitterType<C[K]>
+  ) {
+    let debugElement: DebugElement;
```

Comment:
Please extract it to a `getDebugElement` method and move it to our utils.

Reply:
Not sure where should I put this. There's a method `getNativeElement` at the bottom of this file. Should I create a `getDebugElement` next to it ?

Reply:
Yes